homeassistant/components/mqtt/alarm_control_panel.py

```diff
@@ -81,6 +91,9 @@
 
 PLATFORM_SCHEMA_MODERN = MQTT_BASE_SCHEMA.extend(
     {
+        vol.Required(
+            CONF_SUPPORTED_FEATURES, default=list(_SUPPORTED_FEATURES)
+        ): cv.multi_select(_SUPPORTED_FEATURES),
```

Comment:
The docs PR mentions you can also set a single string. If you want to implement this you should add `cv.ensure_list` in advance to `cv.multi_select()`.